debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -492,15 +508,10 @@ private ZoneId retrieveTransactionTimezone(boolean supportsAtTimeZone) {
         return serverTimezoneConfig == null ? ZoneId.of("UTC") : ZoneId.of(serverTimezoneConfig, ZoneId.SHORT_IDS);
     }
 
-    private String retrieveRealDatabaseName() {
-        try {
-            return queryAndMap(
-                    GET_DATABASE_NAME,
-                    singleResultMapper(rs -> rs.getString(1), "Could not retrieve database name"));
-        }
-        catch (SQLException e) {
-            throw new RuntimeException("Couldn't obtain database name", e);
-        }
+    public String retrieveRealDatabaseName() throws SQLException {
```

Comment:
Could you add a JavaDoc comment to the method which describes why we need to do this? I vaguely remember some potential difference in terms of casing?

Reply:
Done.